sys-auth/authenticator/authenticator-3.32.2.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+PYTHON_REQ_USE="sqlite"
+
+inherit gnome2-utils meson python-single-r1 virtualx xdg-utils
+
+DESCRIPTION="Two-factor authentication code generator for GNOME"
+HOMEPAGE="https://gitlab.gnome.org/World/Authenticator"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.gnome.org/World/Authenticator"
+else
+	SRC_URI="https://gitlab.gnome.org/World/Authenticator/-/archive/${PV}/Authenticator-${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+
+	S="${WORKDIR}/Authenticator-${PV}"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="gnome-shell test"
```

Comment:
`gnome-shell` USE doesn't seem to be having any function in this ebuild, then it shouldn't be used also.

Reply:
it reqs `gnome-base/gnome-shell` for QR code scanning
```bash
[ERROR] 2020-05-15 13:07:27 Could not read the secret code,the keyring keys were reset manually
Traceback (most recent call last):
  File "/usr/lib64/python3.6/site-packages/Authenticator/widgets/accounts/add.py", line 236, in scan_qr
    filename = GNOMEScreenshot.area()
  File "/usr/lib64/python3.6/site-packages/Authenticator/models/screenshot.py", line 56, in area
    -1, None).unpack()
gi.repository.GLib.Error: g-dbus-error-quark: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.gnome.Shell.Screenshot was not provided by any .service files (2)
```

![Selection_102](https://user-images.githubusercontent.com/8144644/82039306-7aaeb500-96ad-11ea-959f-8ec7c1f0010b.png)